cmd/nerdctl/run.go

```diff
@@ -667,6 +667,15 @@ func generateRootfsOpts(ctx context.Context, client *containerd.Client, cliconte
 			containerd.WithNewSnapshot(id, ensured.Image),
 			containerd.WithImageStopSignal(ensured.Image, "SIGTERM"),
 		)
+		for ind, env := range ensured.ImageConfig.Env {
```

Comment:
loop omited when `len(ensured.ImageConfig.Env)=0`

Reply:
yes, I missed this case, I guess I'll need to check on the ENV slice length first before entering the loop....I'll fix it and push the changes,  thanks

Reply:
No need to omit the loop. When the slice is empty, the loop just becomes a nop.